sdk/servicebus/azure-messaging-servicebus-track1-perf/src/main/java/com/azure/messaging/servicebus/perf/App.java

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus.perf;
+
+import com.azure.perf.test.core.PerfStressProgram;
+
+/**
+ * Runs the Storage performance test.
+ *
+ * <p>To run from command line. Package the project into a jar with dependencies via mvn clean package.
+ * Then run the program via java -jar 'compiled-jar-with-dependencies-path' </p>
+ *
+ * <p> To run from IDE, set all the required environment variables in IntelliJ via Run -&gt; EditConfigurations
+ * section.
+ * Then run the App's main method via IDE.</p>
+ */
+public class App {
+    public static void main(String[] args) {
+        Class<?>[] testClasses;
+
+        try {
+            testClasses = new Class<?>[]{
+                Class.forName("com.azure.messaging.servicebus.perf.ReceiveAndDeleteMessageTest"),
```

Comment:
Can we use the full name via ReceiveAndDeleteMessageTest.class.getName()? It'll make refactoring easier.